atom/browser/api/atom_api_auto_updater.cc

```diff
@@ -109,6 +109,7 @@ void AutoUpdater::BuildPrototype(
     v8::Isolate* isolate, v8::Local<v8::ObjectTemplate> prototype) {
   mate::ObjectTemplateBuilder(isolate, prototype)
       .SetMethod("checkForUpdates", &auto_updater::AutoUpdater::CheckForUpdates)
+      .SetMethod("getFeedURL", &auto_updater::AutoUpdater::GetFeedURL)
```

Comment:
@kevinsawicki The autoUpdater on macOS fails to `setFeedURL` when the application is unsigned.  The test would only work for Windows I believe